### PR TITLE
Add auto-generated dkp-cli docs

### PR DESCRIPTION
**Pull request by floridoo**
## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-85041

## Description of changes being made
- a script for generating CLI documentation: `make update-dkp-cli-docs`. By default the current nightly build is used.
- the generated documentation in `pages/dkp/kommander/2.2/cli/` and (symlinked) in `pages/dkp/konvoy/2.2/cli/`
- a Github action that runs daily and opens a PR whenever the generated docs change (upstream changes in the CLIs)

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4160.s3-website-us-west-2.amazonaws.com/

### CODE REVIEW

File: pages/dkp/kommander/2.2/cli/dkp/completion/index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+notes: Automatically generated, DO NOT EDIT
+enterprise: false
+beta: false
+---
+<!-- vale off -->
+<!-- markdownlint-disable -->
+
+## dkp completion
+
+Generate the autocompletion script for the specified shell
```
Comment (cneth) [inline pages/dkp/kommander/2.2/cli/dkp/completion/index.md:16]: I know these are automated, but Is it possible to list the targets of this command (i.e. zsh, fish, etc) here?  I see in other commands we have 'one of [xxxx]',    I notice also that there are help links for all the possibilities, so it seems odd that they are not listed here.

**Review by cneth — APPROVED** [review]
The 'one of [x]' where there is only one item in the list of x is a little awkward, (c.f. "Check a cluster, one of [fips]".   Is it possible to special case this 'one of [list of one item]'?

File: pages/dkp/kommander/2.2/cli/dkp/completion/index.md (hunk 1 above)
Comment (floridoo) [reply pages/dkp/kommander/2.2/cli/dkp/completion/index.md:16]: This is part of the CLI library we use. Might be possible to override though (in the `dkp-cli` repo, will show up here as an automated PR later).

**Review by mwrightd2 — APPROVED** [review]

**Comment by mwrightd2** [comment]
@cneth These files look good from my end, do you have anything youd like to see fixed before I merge?

**Pull request merged**